### ISSUE TITLE
Build ios-integration-test flaky for ios-debug-xcode10

### ISSUE THREAD

**Issue by tobrun** [mapbox/mapbox-gl-native#13061]
I've hit the following issue on CI a couple of times this week when using the `ios-debug-xcode10`:

> MBCameraTransitionTests
  testFlyToCameraInDelegateMethod, Asynchronous wait failed: Exceeded timeout of 5 seconds, with unfulfilled expectations: "Map view should finish loading style.".

Example build [here](https://circleci.com/gh/mapbox/mapbox-gl-native/170427?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link), not sure if this a known issue or not.

cc @mapbox/maps-ios

**Comment by friedbunny** [comment]
Indeed, Xcode 10 on CircleCI has proven to be less than reliable — we’ve bumped timeouts across the board but aren’t sure what the root cause is. The `ios-debug-xcode10` build is still optional, but we’ll need to migrate everything to it sooner rather than later...
